platform/config/config_v2.go

```diff
@@ -341,6 +341,16 @@ func (c *QuesmaNewConfiguration) validatePipelines() error {
 				if queryIndexConf.UseCommonTable != ingestIndexConf.UseCommonTable {
 					return fmt.Errorf("ingest and query processors must have the same configuration of 'useCommonTable' for index '%s' due to current limitations", indexName)
 				}
+				if queryIndexConf.PartitioningStrategy != ingestIndexConf.PartitioningStrategy {
+					return fmt.Errorf("ingest and query processors must have the same configuration of 'partitioningStrategy' for index '%s' due to current limitations", indexName)
+				}
+				if ingestIndexConf.PartitioningStrategy != "" && ingestIndexConf.UseCommonTable { // we don't check for queryIndexConf.PartitioningStrategy per previous check
```

Comment:
Should not we also have similar check for `queryIndexConf`?

Reply:
https://github.com/QuesmaOrg/quesma/pull/1347/commits/3bdef66a7cb6c7b0fdb87e4d9746524c06040e6e